build-scripts/components/gateway-api-crds/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+VERSION="${2}"
+INSTALL="${1}"
+
+mkdir -p "${INSTALL}/k8s/components/charts"
+
+export HELM="$SNAPCRAFT_STAGE/bin/helm"
+
+rm -rf gateway-api-*.tgz
+rm -rf gateway-api
+
+"$HELM" create gateway-api
+rm -rf gateway-api/templates/*
+rm -rf gateway-api/charts
+cp config/crd/standard/* gateway-api/templates/
+cp config/crd/experimental/gateway.networking.k8s.io_tlsroutes.yaml gateway-api/templates/
+sed -i 's/^\(version: \).*$/\1'"${VERSION:1}"'/' gateway-api/Chart.yaml
+sed -i 's/^\(appVersion: \).*$/\1'"${VERSION:1}"'/' gateway-api/Chart.yaml
+"$HELM" package gateway-api
+
+cp gateway-api-*.tgz "${INSTALL}/k8s/components/charts/"
```

Comment:
let's just make this a `build-scripts/update-gateway-api-chart.sh`, which builds the chart and ensures it's in place

it can just call `helm`, and fail if it is not available on the system.